packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -0,0 +1,560 @@
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Upper switch value",
+		"valueSize": 4,
+		"defaultValue": 255
+	},
+	"multilevel_lower_function": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Lower switch value",
+		"valueSize": 4,
+		"defaultValue": 0
+	},
+	"multilevel_duration_function": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Duration",
+		"valueSize": 4
+	},
+	"multilevel_disabled_byte": {
+		"label": "Disable",
+		"valueSize": 4,
+		"minValue": 0,
+		"maxValue": 0,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"button_pressed_threshold_time": {
+		"label": "Pushbutton press threshold time",
+		"unit": "1 ms",
+		"valueSize": 2,
+		"minValue": 1,
+		"maxValue": 1000,
+		"defaultValue": 400,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"button_held_threshold_time": {
+		"label": "Pushbutton held threshold time",
+		"unit": "1 ms",
+		"valueSize": 2,
+		"minValue": 1,
+		"maxValue": 5000,
+		"defaultValue": 600,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": true
+	},
+	"led_indicator_zba7140": {
+		"label": "LED Mode",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 255,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Disable",
+				"value": 0
+			},
+			{
+				"label": "Battery level",
+				"value": 1
+			},
+			{
+				"label": "LED set by parameter 5.",
+				"value": 2
+			}
+		]
+	},
+	"pushbutton_functionality": {
+		"label": "Pushbutton",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Toggle",
+				"value": 0
+			},
+			{
+				"label": "Turn on, dim up",
+				"value": 1
+			},
+			{
+				"label": "Turn off, dim down",
+				"value": 2
+			},
+			{
+				"label": "Momentary",
+				"value": 3
+			}
+		]
+	},
+	"global_brightness": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Global brightness LED",
+		"valueSize": 4,
+		"defaultValue": 255
+	},
+	"color_command_pushbutton": {
+		"label": "Color Commands LED indicator",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Direct control",
+				"value": 0
+			},
+			{
+				"label": "Color for OFF state",
+				"value": 1
+			},
+			{
+				"label": "Color for ON state",
+				"value": 2
+			}
+		]
+	},
+	"dimmer_pushbutton": {
+		"$import": "../../templates/master_template.json#base_0-255_nounit",
+		"label": "Pushbutton used for dimmer",
+		"maxValue": 15
```

Comment:
Is there an explanation for the values? 15 makes me suspicious that this should be partial parameters instead.

Reply:
This can be removed from the code. references overlapping bit masks